<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestForm</class>
 <widget class="QWidget" name="TestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonBrowseTemp">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>60</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBrowseTemp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>模板文件路径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBrowseTemp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBrowseMatch">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>160</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBrowseMatch">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelBrowseMatch">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Match文件路径：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStartMatch">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>开始比对</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBrowseEnroll">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Match文件路径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBrowseEnroll">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBrowseEnroll">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>400</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览并生成</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
